src/main/java/frc/robot/subsystems/ampbar/AmpBar.java

```diff
@@ -7,18 +7,40 @@
 
 package frc.robot.subsystems.ampbar;
 
+import static edu.wpi.first.units.Units.Seconds;
+import static edu.wpi.first.units.Units.Volts;
+
+import com.ctre.phoenix6.SignalLogger;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.StartEndCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
 import org.littletonrobotics.junction.Logger;
 
 public class AmpBar extends SubsystemBase {
 
   private final AmpBarIO ampBarIO;
   private final AmpBarIOInputsAutoLogged ampBarIOAutoLogged = new AmpBarIOInputsAutoLogged();
 
+  private final SysIdRoutine m_sysIdRoutine;
+
   public AmpBar(AmpBarIO ampBarIO) {
     this.ampBarIO = ampBarIO;
+    m_sysIdRoutine =
```

Comment:
ampbar is velocity now?

Reply:
the sysID routine is the same for velocity and position

Reply:
oh right i meant its closed-loop now?

Reply:
eventually it should be, cuz the amp bar kinda slow and bouncy